src/style.css

```
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.searchForm {
+  margin: 20px 0;
+}
+
+.searchBar {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  display: flex;
+  width: 100%;
+
+  font-family: inherit;
+
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchBar::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #404e5f;
+}
+
+.big {
```

Avoid using this type of naming. It's no clear what is actually big.

```suggestion
.searchBar--big {
```

or `.big-searchbar`